### ISSUE TITLE
Something's broken with blog posts and manual date

### ISSUE THREAD

**Issue by thenktor** [typemill/typemill#259]
Hi,

there is something broken with blog articles and manual date. New blog articles have file name like this (date+time at the beginning):
```
bash-4.3# ls -l 00-blog/
total 24
-rw-r--r-- 1 http http  30 Jun  1 14:06 202106011406-test-blog-entry.md
-rw-r--r-- 1 http http 203 Jun  1 14:06 202106011406-test-blog-entry.yaml
-rw-r--r-- 1 http http  28 Jun  1 14:07 202106011407-another-article.md
-rw-r--r-- 1 http http 199 Jun  1 14:07 202106011407-another-article.yaml
-rw-r--r-- 1 http http  17 Jun  1 13:56 index.md
-rw-r--r-- 1 http http 273 Jun  1 14:05 index.yaml
```

After changing the manual date of an article to 1980-04-10:
```
bash-4.3# ls -l 00-blog/
total 24
-rw-r--r-- 1 http http  28 Jun  1 14:07 198004101409-another-article.md
-rw-r--r-- 1 http http 287 Jun  1 14:09 198004101409-another-article.yaml
-rw-r--r-- 1 http http  30 Jun  1 14:06 202106011406-test-blog-entry.md
-rw-r--r-- 1 http http 203 Jun  1 14:06 202106011406-test-blog-entry.yaml
-rw-r--r-- 1 http http  17 Jun  1 13:56 index.md
-rw-r--r-- 1 http http 273 Jun  1 14:05 index.yaml
```

After removing the manual date from this article again the date disappears, but seconds are added:
```
bash-4.3# ls -l 00-blog/
total 24
-rw-r--r-- 1 http http  28 Jun  1 14:07 141145-another-article.md
-rw-r--r-- 1 http http 277 Jun  1 14:11 141145-another-article.yaml
-rw-r--r-- 1 http http  30 Jun  1 14:06 202106011406-test-blog-entry.md
-rw-r--r-- 1 http http 203 Jun  1 14:06 202106011406-test-blog-entry.yaml
-rw-r--r-- 1 http http  17 Jun  1 13:56 index.md
-rw-r--r-- 1 http http 273 Jun  1 14:05 index.yaml
```

Also the folder type of the *blog* folder changes to pages instead of posts.

**Comment by trendschau** [comment]
good find, thank you, I have to check that for next release

**Comment by trendschau** [comment]
fixed it for release 1.4.6. which will probably release next week.

**Comment by trendschau** [comment]
release 1.4.6 is published now
